extensions/control-plane/api/management-api/contract-negotiation-api/src/test/java/org/eclipse/edc/connector/api/management/contractnegotiation/ContractNegotiationApiControllerIntegrationTest.java

```diff
@@ -29,6 +29,7 @@
 import org.eclipse.edc.spi.message.RemoteMessageDispatcher;
 import org.eclipse.edc.spi.message.RemoteMessageDispatcherRegistry;
 import org.eclipse.edc.spi.query.SortOrder;
+import org.eclipse.edc.spi.types.domain.callback.CallbackAddress;
```

Comment:
IMO this test and possibly also the `ContractNegotiationApiControllerTest` should explicitly verify these cases:
- with _one_ callback address
- with _no_ callback address
- with _multple_ callback addresses. 

could be a `ParameterizedTest`